product-mini/platforms/nuttx/wamr.mk

```diff
@@ -268,8 +268,14 @@ endif
 
 ifeq ($(CONFIG_INTERPRETERS_WAMR_DISABLE_HW_BOUND_CHECK),y)
 CFLAGS += -DWASM_DISABLE_HW_BOUND_CHECK=1
+CFLAGS += -DWASM_DISABLE_STACK_HW_BOUND_CHECK=1
 else
 CFLAGS += -DWASM_DISABLE_HW_BOUND_CHECK=0
+  ifeq ($(CONFIG_INTERPRETERS_WAMR_DISABLE_STACK_HW_BOUND_CHECK),y)
```

Comment:
No such option in NuttX now，we can disable it if CONFIG_INTERPRETERS_WAMR_DISABLE_HW_BOUND_CHECK disabled.
BTW, even CONFIG_INTERPRETERS_WAMR_DISABLE_HW_BOUND_CHECK exists but it doesn't works actually since most NuttX supported platform don't have MMU.